### DISCUSSION TITLE
Add FilterGrep in Banzai Logging FluentbitSpec

### DISCUSSION THREAD

**Discussion by ltartarini** [kube-logging/logging-operator#1642] [Q&A] [answered]
Hi *,

Is there any chance to have the FluentBit Filter Grep (https://docs.fluentbit.io/manual/pipeline/filters/grep) into the FluentbitSpec of the Banzai Logging module? So far I see only the following FluentBit filters (https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types):
- [FilterKubernetes](https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types/#filterkubernetes)
- [FilterAws](https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types/#filteraws)
- [FilterModify](https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types/#filtermodify)
- [FilterModifyRule](https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types/#filtermodifyrule)
- [FilterModifyCondition](https://kube-logging.dev/docs/configuration/crds/v1beta1/fluentbit_types/#filtermodifycondition)

It can be helpful, for example, to exclude logs before passing them to Fluentd.

Thanks in advance, 

Luca Tartarini

**Comment by pepov** [comment, marked as answer]
Hey @ltartarini ! Yes of course. I recommend creating an issue for this and the most efficient way to get it done if you could create a PR. We are happy to help contributors get up and running over the project discord https://discord.gg/UXcKU7GHXT
Otherwise the contribution process is straightforward, you can read about it here: https://github.com/kube-logging/logging-operator/blob/master/CONTRIBUTING.md

**Reply by ltartarini** [reply]
Opened here: https://github.com/kube-logging/logging-operator/issues/1647
